avocado_vt/plugins/vt.py

```diff
@@ -42,59 +41,109 @@
                       "plugin to get rid of this message")
 
 
+def add_option(parser, namespace, arg, default=None, action='store',
+               help_msg='', nargs=None):
+    if hasattr(Settings, 'add_argparser_to_option'):
```

Comment:
`virttest.compat` seems an appropriate place for this utility, because there's already a utility for checking if the settings is "new or old" style (`is_registering_settings_required()`), and also a conditional block in which you can add two versions of this `add_option()` utility.  There's already `get_opt()`, `set_opt()`, `set_opt_from_settings()` and `get_settings_value()` there.

Reply:
Cool, I didn't know about this. I will use it